File: 8-kyu/reversed_words/reversed_words.go
package reversed_words

import (
	"strings"
)

func ReverseWords(str string) string {
	ss := strings.Fields(str)
	for i, j := 0, len(ss)-1; i < j; i, j = i+1, j-1 {
		ss[i], ss[j] = ss[j], ss[i]
	}
	return strings.Join(ss, " ")
}

File: 8-kyu/reversed_words/reversed_words.py
def reverse_words1(s):
    words = s.split()
    i = 0; j = len(words) -1
    while i<j:
        words[i], words[j] = words[j], words[i]
        i+=1; j-=1
    return " ".join(words)

def reverse_words2(s):
    words = s.split()
    reversedWords = words[::-1]
    return " ".join(reversedWords)

def reverse_words3(s):
    words = s.split()
    return " ".join(reversed(words))